.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,9 @@ approved by the reviewer.
 
 Once the pull request is approved, a team member will take care of merging.
 
+## Contributing models
+When contributing new models, please validate model performance by providing training results. You can do this using our existing [ImageNet training script](https://github.com/keras-team/keras-cv/blob/master/examples/training/classification/imagenet/basic_training.py) or by contributing a custom training script of your own (see "Contributing training scripts" below). Training results can be added to the training history log with [this script](https://github.com/keras-team/keras-cv/blob/master/shell/weights/update_training_history.py), or shared with the team via Google Drive (we'll need TensorBoard logs as well as weights). Either way, the KerasCV team will need to upload the weights to our GCS bucket for distribution.
```

Comment:
...
In particular, we don't expect the model converges to paper claimed result in the first PR, but we encourage contributors to keep polishing training scripts until it reaches on par result and even surpass it with better augmentation, regularization, learning rate schedule, longer epochs, etc.

Reply:
I've added some specification about this.